### DISCUSSION TITLE
WARN] - (starship::modules::time): Error in module `time`

### DISCUSSION THREAD

**Discussion by thamaraiselvam** [starship/starship#1850] [Q&A] [answered]
Suddenly getting this error 

```

[WARN] - (starship::modules::time): Error in module `time`:
 --> 1:18
  |
1 | [ %I:%M:%S%.3f %p]
  |                  ^---
  |
  = expected variable, string, textgroup, or conditional
```

```
 python --version
Python 3.8.2
```

```
starship --version
starship 0.46.2
```

**Comment by andytom** [comment, marked as answer]
Hi @thamaraiselvam, I think this is going to be a problem with the configuration you are using for the time module. Would it be possible for you to share your `starship.toml` configuration file?

**Reply by rlabrecque** [reply]
Hey @andytom I started hitting this as well. Same config on a new PC:

```
❯ starship --version
starship 0.50.0
tag:v0.50.0
commit_hash:0558b38e
build_time:2021-02-02 20:04:56
build_env:rustc 1.49.0 (e1884a8e3 2020-12-29),
```

```
[time]
disabled = false
use_12hr = true
format = "[ %T ]"

```

**Reply by rlabrecque** [reply]
Updating it to:

```
[time]
disabled = false
use_12hr = true
format = '[\[ $time \]]($style) '
time_format = "%T"
```

Via the latest documentation fixed it for me 👍

**Reply by andytom** [reply]
Ah ok looks like this is going to be caused by the [v0.45.0 migration](https://starship.rs/migrating-to-0.45.0/), thanks for the update @rlabrecque
